test/gui/shared/steps/steps.py

```diff
@@ -30,29 +31,19 @@
 confFilePath = confdir + 'owncloud.cfg'
 socketConnect = None
 
-passwords = {
-    'alt1': '1234',
-    'alt2': 'AaBb2Cc3Dd4',
-    'alt3': 'aVeryLongPassword42TheMeaningOfLife',
-}
-
-defaultUsers = {
-    'Alice': {
-        'displayname': 'Alice Hansen',
-        'password': passwords['alt1'],
-        'email': 'alice@example.org',
-    },
-    'Brian': {
-        'displayname': 'Brian Murphy',
-        'password': passwords['alt2'],
-        'email': 'brian@example.org',
-    },
-    'Carol': {
-        'displayname': 'Carol King',
-        'password': passwords['alt3'],
-        'email': 'carol@example.org',
-    },
-}
+defaultUsers = None
+
+
+def sendRequestToGetMiddlewareState(context):
+    res = requests.get(
+        os.path.join(context.userData['middlewareUrl'], 'state'),
+        headers={"Content-Type": "application/json"},
+    )
+    try:
+        global defaultUsers
```

Comment:
is the middleware returning "default" users or the really created users? If there are the users that have been created during the test runs, we maybe should not cache them here, because they might change

Reply:
It returns created users. In cases where users are created during test runs like in When step we need to fetch the middleware endpoint again.